db.go

```diff
@@ -110,6 +110,12 @@ type DB struct {
 	// of truncate() and fsync() when growing the data file.
 	AllocSize int
 
+	// MaxSize is the maximum size (in bytes) allowed for the data file.
+	// If a caller's attempt to add data results in the need to grow
+	// the data file, an error will be returned and the data file will not grow.
+	// 0 means no limit.
+	MaxSize int
```

Comment:
`int` means that users can pass in a negative value. Probably we should,
- change the comment "0 means no limit" to "0 or a negative value means no limit"
- change the if condition "if db.MaxSize != 0" to "if db.MaxSize > 0"

Reply:
I've made this change